accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -217,7 +220,16 @@ assignable to `Object`.
 
 It is not an error for the body of a `late` field to reference `this`.
 
-It is an error for a formal parameter to be declared `late`.
+It is an error for a variable to be declared as `late` in any of the following
+positions: in a formal parameter list of any kind; in a catch clause; in the
```

Comment:
@leafpetersen where exactly a catch clause use of 'late' is an error?
```dart
} on Exception catch (late1 e) {
  late2 int i;
}
```
Does this sentence disallow 'late' in late1 or late2 positions?

Reply:
The catch clause actually doesn't allow for this syntax, it's `<identifier> (',' <identifier>)`, so `late1 e` is just a syntax error.

The position `late2` must be allowed, that's like any other block of code. We might want to omit the catch clause from the above sentence and just mention in commentary that a `<catchPart>` introduces a local variable or two, but doesn't syntactically allow for `late`.